epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -231,7 +233,57 @@ public int hashCode() {
   }
 
   boolean isViewClickListener() {
-    return isViewClickListenerType(getTypeMirror()) || isViewLongClickListenerType(getTypeMirror());
+    return isViewClickListenerType(getTypeMirror());
+  }
+
+  boolean isViewLongClickListener() {
+    return isViewLongClickListenerType(getTypeMirror());
+  }
+
+  boolean isBoolean() {
+    return getTypeMirror().getKind() == TypeKind.BOOLEAN;
+  }
+
+  boolean isCharSequenceOrString() {
+    return Utils.isCharSequenceOrStringType(getTypeMirror());
+  }
+
+  boolean isStringList() {
+    return Utils.isType(getTypeMirror(), "java.util.List<java.lang.String>")
+        || Utils.isType(getTypeMirror(), "java.util.List<kotlin.String>");
+  }
+
+  boolean isEpoxyModelList() {
+    return Utils.isType(
+        getTypeMirror(),
+        "java.util.List<? extends com.airbnb.epoxy.EpoxyModel<?>>"
```

Comment:
would be nice to use a ClassName constant or something, not sure if that is possible with the wildcard